NEWS.md

```diff
@@ -1,3 +1,8 @@
+DALEX 2.1.0
+---------------------------------------------------------------
+* Added new parameter (`predict_function_column`) to `explain` function that allows specifying positive class in binary classification tasks ([#250](https://github.com/ModelOriented/DALEX/issues/250)).
```

Comment:
thanks, it would be great to add either some examples with this feature or a vignette

Reply:
Good idea, I will rebuild multiclass vignette in another issue